src/test/java/org/cactoos/time/DateOfTest.java

```diff
@@ -73,6 +73,24 @@ public final void testParsingCustomFormattedStringToDate() {
         ).affirm();
     }
 
+    @Test
+    public final void testParsingCustomFormattedStringWithoutTimeToDate() {
+        new Assertion<>(
+            "Can't parse a Date with custom format.",
```

Comment:
I'm confused with the negative clause of the assertion. I see that it's kinda an error message stub for a failed test case, but we have description of an intent in the other tests. @llorllale do we use `Must/should do something` statement or `Can't do something`?

Reply:
@atapin good point, I always struggle with that. Usually I use the "must" form, but here I copy-pasted it from another tests and didn't change the message. Let's see what @llorllale says but I propose to change the wording with "must" (as in `Assertion` javadoc).

Reply:
@atapin you are 100% correct, ie use a positive clause starting with a modal verb like "must"

Reply:
@victornoel can we fix unrelated test cases in the scope of the current PR?